core/src/wallet/tezos/explorers/api/TezosLikeTransactionParser.cpp

```diff
@@ -183,6 +177,12 @@ namespace ledger {
                     }
                 } else if (_lastKey == "public_key") {
                     _transaction->publicKey = value;
+                } if (_lastKey == "amount") {
```

Comment:
That `if` will always be tested, even if a previous branch has succeeded. I think you need an `else if`.

Reply:
Clearly a typo ! Good catch ! 👍